projects/focil-geth-and-prysm-poc.md

```diff
@@ -0,0 +1,80 @@
+# Fork-Choice Enforced Inclusion Lists (FOCIL) Proof of Concept in Geth and Prysm
+
+Enhancing Decentralization and Censorship Resistance in Block Production
```

Comment:
Here or in the goal of the project I would note that it's meant as prototyping for feasibility research so it's clear it's not an EIP getting to mainnet

Reply:
sure.